azure/services/async/async.go

```diff
@@ -29,6 +32,9 @@ import (
 	"sigs.k8s.io/cluster-api-provider-azure/util/tele"
 )
 
+// defaultHTTP429RetryAfter is a default backoff wait time when we get a HTTP 429 response with no Retry-After data.
+const defaultHTTP429RetryAfter = 1 * time.Minute
```

Comment:
can we move this const to `reconciler/defaults.go` where the other requeue consts are defined?

Reply:
done